### DISCUSSION TITLE
写笔记的时候出现闪退

### DISCUSSION THREAD

**Discussion by ghost** [windingwind/zotero-better-notes#762] [Q&A] [answered]
闪退逻辑是：当采用latex语法写一个公式，比如$y=Ax+b$，得到了 $y=Ax+b$ ，此时将光标放到 $b$ 后面进行输入时，偶尔出现闪退。今天（25Oct2023）已经触发多次该问题。求助

**Comment by windingwind** [comment, marked as answer]
与插件无关向官方论坛反馈

**Reply by ghost** [reply]
多谢大大回复，我今天排查了一下，只有在note内容很长的情况下会出现闪退问题。已经把长笔记拆开来了
